functions/postgrest.sql

```diff
@@ -6,8 +6,10 @@ BEGIN
         ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT, UPDATE, DELETE, INSERT ON TABLES TO postgrest_api;
         GRANT USAGE, SELECT ON ALL SEQUENCES IN SCHEMA public TO postgrest_api;
     END IF;
+    ALTER ROLE postgrest_api SET statement_timeout = '60s';
```

Comment:
This needs to be configurable, and be updated even after the role has been created